src/isError.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * @typedef {'SNAKE' | 'KEBAB' | 'CAMEL' | 'PASCAL' | 'UPPER'} CaseName
+ *
+ * @param {CaseName | undefined} param
+ * @param {string} pathname
+ *
+ * @returns {Array<{ message: string }> | undefined}
+ */
+
+function isError(param, pathname) {
+  const errors = [];
+  const isAvailable = ['SNAKE' , 'KEBAB' , 'CAMEL' , 'PASCAL' , 'UPPER'].some((toCase) => toCase === param);
+
+  if(!pathname) {
+    errors.push({
+      message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`
```

Comment:
Will be great to move the messages for the different errors to the constants too